src/views/FlowPostToConversation.vue

```diff
@@ -5,17 +5,17 @@
 
 <template>
 	<div>
-		<NcSelect :value="currentRoom"
+		<NcSelect :model-value="currentRoom"
```

Comment:
[Vue3] ReferenceError: currentRoom is undefined

Reply:
Looks fine by the code. Do you mean TS error or runtime?

Reply:
doesn't work in runtime, but could be fixed later